### DISCUSSION TITLE
Install personal instance

### DISCUSSION THREAD

**Discussion by dawidol** [yume-chan/ya-webadb#469] [Q&A] [answered]
Hi

I am looking for instructions, guides and/or tutorials on how to install my own instance of ya-webadb.

Any help is welcome :)

thanks.

**Comment by yume-chan** [comment, marked as answer]
## Deploy to GitHub Pages:

1. Clone this repo
2. Enable actions from settings

![image](https://user-images.githubusercontent.com/1330321/209154110-b6f30ed8-795c-41b3-a57d-5b38effb8ee5.png)

## Deploy to somewhere else:

See https://github.com/yume-chan/ya-webadb/blob/main/CONTRIBUTING.md

**Comment by dawidol** [comment]
Awesome, that work on my Raspberry Pi :) Thank you, amazing work
